NavigateU/NavigateU/LoginScreen/LoginViewController.swift

```diff
@@ -21,4 +28,28 @@ class LoginViewController: UIViewController {
         view.backgroundColor = .white
     }
 
+    private func gotoHomePage() {
+        let viewController = MainScreenViewController()
+        self.navigationController?.pushViewController(viewController, animated: true)
+    }
+    @objc private func loginButtonTapped() {
+        print("Login button tapped")
+        guard let email = loginDesignview?.email.text, let password = loginDesignview?.password.text else {
+            print("Please enter email and password")
+            return
+        }
+            viewModel.login(email: email, password: password)
+            configureIO()
+
+    }
+    private func configureIO() {
+        viewModel.$error.sink { [weak self] error in
+            if let error = error {
+                print(error)
```

Comment:
you can make an extension to UIViewController, that will show notificationBanner for all your errors, print is a bad practice at user experience, but okay for temporary solution